packages/redux/src/settings/reducer/setting.ts

```diff
@@ -0,0 +1,69 @@
+import * as actionTypes from '../actionTypes.js';
+import { type AnyAction, combineReducers, type Reducer } from 'redux';
+import type { BlackoutError } from '@farfetch/blackout-client';
+import type { SettingState } from '../types/index.js';
+
+export const INITIAL_STATE: SettingState = {
+  error: null,
```

Comment:
Can the user request multiple settings in a typical session?
If so, then I would index these values by setting id.

Reply:
Changed it 👍